apps/aehttpserver/web/controllers/block_controller.ex

```diff
@@ -18,6 +18,29 @@ defmodule Aehttpserver.BlockController do
     end
   end
 
+  def get_blocks(conn, params) do
+    latest_block_hash = case Map.get(params, "from_block") do
+      nil ->
+        latest_block = Chain.latest_block()
+        BlockValidation.block_header_hash(latest_block.header)
+      hash ->
+        {_, hash_bin} = Base.decode16(hash)
+        hash_bin
+    end
+
+    count = case Map.get(params, "count") do
+      nil ->
+        100
+      count_string ->
+        {number, _} = Integer.parse(count_string)
```

Comment:
We could add an limit here.

Reply:
I think it's ok for now, we have to optimize this anyway in the future (e.g. only give header)